process/support_team.md

```diff
@@ -4,43 +4,47 @@
-
-Once an issue is complete, the customer should be notified and the issue moved to the 'Done' column for the current week. 
-
-## Working in Support
-
-If you're working on support in a week (or part of a week) then you should:
- * Attend the planning session at the start of the week, please come armed with the number of days you can commit to support that week.
- * Attend stand-ups on days you are doing support work.
- * Where possible we like to pair on issues, especially for our more esoteric projects, in order to spread domain knowledge throughout the company, so come willing and able to do so.
+## Support Planning
 
-### The Triage Team
+During a weekly call (bi-weekly for some) with each customer Nick and the customer will agree to a commitment of work to be completed the following week by the support team. These issues will be moved to the 'Up Next' column, and copied to the Support Epics board.
 
-As part of your support week you might want to volunteer to be on the triage team. This generally consists of one of Nick, Luke, Chris, Rory plus one other person.
+By Thursday of each week the Support Epics board will have:
+ * A list of items that can be reasonably completed by support pairs in the upcoming week, and have a pair assigned to them.
+ * A list of 'Triage Team' items, which can reasonably be completed by memebers of the Triage Team around core Triage work.
 
-The Triage Team rotates every week, and it's their responsibility to ensure that anything new in Groove is resolved within 48 hours (noting that moving to backlog is a suitable resolution for non-urgent issues).
+## Working in a Support Pair
 
-Note that any new activity on Groove triggers a notification to the #support channel in Slack, so it's worth idling in there to save having to keep hitting refresh...
+If you're on a Support Pair then it is your responsibility to:
+ * Ensure you know at the start of each week what your pair will be working on, and have a goal in mind for the Scrum of Scrums.
+ * Book in at least one showcase with the customer that week so they can see what progress ahs been made towards the goal.
+ * At the end of each day record your time in Harvest against the relevant customer. You should also include a brief description of the work carried out.
+ * Communicate regulalarly with the customer especially around deployments so they can plan any testing they may want to do.
 
+## Working in Team Triage
 
+If you're working on Team Triage then it is your responsibility to:
+ * Know which days you are on Triage duty, and swap days in advance if you're unable to.
+ * Work towards the Triage goal of resolving everything within 48 hours (NB. Backlogging an issue is an accepted resolution)
+ * Call for help from others if there is more triage work than you can resonably handle
```

Comment:
What is the internal mechanism which ensures this doesn't have a negative impact on other customer goals?